src/main/resources/ErrorMessage.properties

```diff
@@ -0,0 +1,4 @@
+ERROR_MESSAGE_TEMPORARY_BUCKET=Bucket name can only contain lowercase letters, numbers, '.', '_', and '-'.
```

Comment:
This is very specific to google-cloud and should live in google-cloud repo. Only common stuff that can be reused in other plugins should go into this repo

Reply:
moved to google-cloud